src/Controller/Api/FilesController.php

```diff
@@ -61,32 +61,28 @@ public function getFiles(int $id)
     }
```

Comment:
Maybe this should be the uploadController?

Reply:
Yes, Makes sense to rename it